scss/_mixins.scss

```diff
@@ -42,19 +42,7 @@
     .h4,
     .h5,
     .h6 {
```

Comment:
May be able to replace lines 32 - 44 with a * , but not sure if that is too heavy handed.

Reply:
I don't think * is necessary. Even p on lines 32 and 51 don't seem necessary. It's only the headers that are explicilty changed in _typography.